### DISCUSSION TITLE
Grid size

### DISCUSSION THREAD

**Discussion by NBarile** [TomSchimansky/CustomTkinter#1045] [Q&A]
Hi everyone ! 

I have a simple question about my app : 
![image](https://user-images.githubusercontent.com/48790575/212128026-200e603c-af3f-4f2a-853f-cfd30372b7ed.png)

Here, I want my white frame to take up all the rest of the gray frame. How can I do ?
I use the .grd() method to do it, and I tried colomnspan but it dosen't work. 

Here is my code : 
```PYTHON
self.button = ctk.CTkFrame(master=self.info_label, bg_color='white')
self.button.grid(row=0, column=0, padx=10, pady=10, sticky="nsew")
self.telemetry = ctk.CTkFrame(master=self.info_label, bg_color='white')
self.telemetry.grid(row=0, column=1, padx=10, pady=10, sticky="nsew")
```

Thanks :D

**Comment by Akascape** [comment]
@NBarile Try messing with the `sticky`.
